packages/web/src/scss/theme/_form-fields.scss

```diff
@@ -1,56 +1,63 @@
-@use '@tokens' as tokens;
+@use '@global' as global-tokens;
+@use '../settings/globals';
 
 // Common for all form components
-$gap: tokens.$space-400;
-$input-typography: tokens.$body-medium-text-regular;
-$input-color-disabled: tokens.$text-primary-disabled;
-$input-border-color-success: tokens.$emotion-success-default;
-$input-border-color-warning: tokens.$emotion-warning-default;
-$input-border-color-danger: tokens.$emotion-danger-default;
-$input-focus-shadow: tokens.$focus;
-$label-color-default: tokens.$text-primary-default;
-$label-color-disabled: tokens.$text-primary-disabled;
-$label-required-margin-left: tokens.$space-200;
-$label-required-color: tokens.$emotion-danger-default;
-$validation-text-typography: tokens.$body-medium-text-regular;
-$validation-text-margin-top: tokens.$space-300;
-$validation-text-color-default: tokens.$text-secondary-default;
-$validation-text-color-disabled: tokens.$text-secondary-disabled;
-$validation-text-color-success: tokens.$emotion-success-default;
-$validation-text-color-warning: tokens.$emotion-warning-default;
-$validation-text-color-danger: tokens.$emotion-danger-default;
-$helper-text-typography: tokens.$body-small-text-regular;
-$helper-text-margin-top: tokens.$space-300;
-$helper-text-color-default: tokens.$text-secondary-default;
-$helper-text-color-disabled: tokens.$text-secondary-disabled;
+$gap: global-tokens.$space-500;
+$input-typography: global-tokens.$body-medium-regular;
+$input-color-disabled: var(--#{globals.$prefix}color-disabled-content);
+$input-background-color-disabled: var(--#{globals.$prefix}color-disabled-background);
+$input-border-color-success: var(--#{globals.$prefix}color-form-field-success);
+$input-border-color-warning: var(--#{globals.$prefix}color-form-field-warning);
+$input-border-color-danger: var(--#{globals.$prefix}color-form-field-danger);
+$input-border-color-disabled: var(--#{globals.$prefix}color-disabled-border);
+$input-focus-shadow: global-tokens.$focus;
+$label-color-default: var(--#{globals.$prefix}color-form-field-label);
+$label-color-disabled: var(--#{globals.$prefix}color-disabled-content);
+$label-required-margin-left: global-tokens.$space-200;
+$label-required-color: var(--#{globals.$prefix}color-form-field-danger);
+$validation-text-typography: global-tokens.$body-small-semibold;
```

Comment:
The asterisk is just `regular` in Figma:

![image](https://github.com/user-attachments/assets/421f3808-acf0-48e8-a3da-fd71c3e0687d)